skiko/src/awtMain/kotlin/org/jetbrains/skiko/FontInterop.awt.kt

```diff
@@ -0,0 +1,118 @@
+package org.jetbrains.skiko
+
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.launch
+import java.awt.Font
+import java.awt.FontFormatException
+import java.awt.GraphicsEnvironment
+import java.io.File
+import java.io.FileInputStream
+import java.io.IOException
+import java.util.concurrent.ConcurrentHashMap
+
+object AwtFontManager {
+    private var fontsMap = ConcurrentHashMap<String, File>()
+    @Volatile
+    private var allFontsCachedImpl = false
+
+    private val waitChannel = RendezvousBroadcastChannel<Int>()
+
+    init {
+        GlobalScope.launch(Dispatchers.IO) {
+            cacheSystemFonts()
```

Comment:
Because of this, we probably can't quickly exit the application (Compose uses `exitProcess`, but we shouldn't rely on it).

So, I suggest to move it out of the API behavior too (see the previous comment).

Or add `yield` after every font load, so coroutine can quickly exit on cancellation.

Reply:
`yield()` idea sounds right, done it.